### PR TITLE
JDTP-21 - Create UI components for display selected chart screen

### PR DESCRIPTION
**Pull request by asprincean**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@craco/craco": "^6.4.5",
```
Comment (rellisbrown) [inline package.json]: Lol, how did this get in here?

**Review by rellisbrown — APPROVED** [review]
Looks really good!

File: package.json (hunk 1 above)
Comment (asprincean) [reply package.json]: my fault......it wasn't working npm install

Comment (asprincean) [reply package.json]: how I can fix that?

Comment (rellisbrown) [reply package.json]: Oh right, so its an issue with the dependencies of the nebula stardust package (and I think all of the individual chart packages too). If you run npm install --legacy-peer-deps it should work

Comment (asprincean) [reply package.json]: thank you

**Review by michaelcairney — APPROVED** [review]

**Pull request merged**